modules/mfa/src/Auth/Process/Mfa.php

```diff
@@ -808,6 +810,33 @@ public static function setRememberMeCookies(
         setcookie('c2', $expireDate, $expireDate, '/', null, $secureCookie, true);
     }
 
+    /**
+     * Clear remember_me cookies (c1 and c2)
+     */
+    public static function clearRememberMeCookies(): void
+    {
+        $secureCookie = Env::get('SECURE_COOKIE', true);
+        setcookie('c1', '', time() - 3600, '/', null, $secureCookie, true);
```

Comment:
just curious why you used `time() - 3600` and not just `time()`

Reply:
I guess it's considered best practice by some on [stacko](https://stackoverflow.com/questions/56722961/is-setting-cookie-expiration-time-time-3600-always-correct)

Reply:
It's even in the [docs](https://www.php.net/manual/en/function.setcookie.php)

Reply:
Kinda seems like a "we've always done it that way" sort of thing. I can't figure out how it helps. If it's a time zone issue, shouldn't it be `time() - 3600 * 24`?

Reply:
You're probably right, but time drift of the client is also mentioned. Not sure how common that is these days.